.github/workflows/ci.yaml

```diff
@@ -0,0 +1,55 @@
+name: openEQUELLA Admin Console Launcer CI
+
+on:
+  push:
+    branches:
+      - master
+      - develop
+      - release/**
+    tags:
+      - "**"
+  pull_request:
+    branches:
+      - master
+      - develop
+      - release/**
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Gradle Cache
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-
+
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v2
+        with:
+          distribution: adopt
+          java-version: 8
+
+      - name: Build with Gradle
+        run: ./gradlew build
```

Comment:
Does the `build` task also run the full test suite? (I think it does, just want to confirm)

Reply:
Yes I think so.

> By default, Gradle will run all tests that it detects, which it does by inspecting the compiled test classes. This detection uses different criteria depending on the test framework used.

Reply:
I confirmed by looking at the build output and it did report compile test classes and running tests.

That said, unfortunately there is only one test. :'(